big_tests/tests/cets_disco_SUITE.erl

```diff
@@ -144,24 +157,82 @@ rdbms_backend_db_queries(_Config) ->
     TS2 = TS + 100,
 
     %% insertion fails if node name or node num is already added for the cluster
-    ?assertEqual({updated, 1}, insert_new(CN, <<"test1">>, TS, 1)),
-    ?assertMatch({error, _}, insert_new(CN, <<"test1">>, TS, 1)),
-    ?assertMatch({error, _}, insert_new(CN, <<"test1">>, TS, 2)),
-    ?assertMatch({error, _}, insert_new(CN, <<"test2">>, TS, 1)),
-    ?assertEqual({updated, 1}, insert_new(CN, <<"test2">>, TS, 2)),
+    ?assertEqual({updated, 1}, insert_new(CN, <<"test1">>, 1, <<>>, TS)),
```

Comment:
For me it would be more natural to have NULL used in this filed. Otherwise we have a field that can be either NULL or an empty string.